.gitignore

```diff
@@ -12,3 +12,4 @@ composer.phar
 var/
 composer.lock
 .php_cs.cache
+.idea
```

Comment:
@pawbuj we don't exclude developer environment-specific files in .gitignore. Please use global gitignore to exclude PhpStorm directory. I keep mine in `.config/git/ignore`, but that might be OS-dependent. Here's [GitHub recommendation](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) how to do it.

Reply:
ok